lib/components/app/app-menu.js

```diff
@@ -52,47 +142,69 @@ class AppMenu extends Component {
       toggleMailables
     } = this.props
 
+    const { isPaneOpen } = this.state
     return (
-      <div aria-label='app-menu' className='app-menu' role='navigation'>
-        <DropdownButton
-          aria-label='Application Menu'
-          className='app-menu-button'
-          id='app-menu'
-          noCaret
-          title={(<Icon type='bars' />)}>
-          <MenuItem onClick={this._showRouteViewer}>
-            <Icon type='bus' /> {languageConfig.routeViewer || 'Route Viewer'}
-          </MenuItem>
-          {callTakerEnabled &&
-            <MenuItem onClick={resetAndToggleCallHistory}>
-              <Icon type='history' /> Call History
-            </MenuItem>
-          }
-          {fieldTripEnabled &&
-            <MenuItem onClick={resetAndToggleFieldTrips}>
-              <Icon type='graduation-cap' /> Field Trip
+      <>
+        <div
+          aria-label={isPaneOpen ? 'Close Menu' : 'Open Menu'}
```

Comment:
Now that localization is well under way and that this PR depends on #435, I'm going to recommend to extract this string to the i18n YML files.

Reply:
Prep added in 542d756098ceae9ad56f74334fae242b4be7c938